Documentation/index.rst

```diff
@@ -23,12 +25,9 @@ Configuration requires some setup on the side of Google, before the package can
 In a nutshell:
 
 #. Log in to the Google Developers Console
-#. Set up the Consent screen
-#. Create a web application Client ID
-#. Create a public access Server key
+#. Create a service account key
```

Comment:
You also have to add permissions for the created service account in your Google Analytics account.